api.ts

```diff
@@ -151,7 +151,7 @@ export interface SourceFileConfiguration {
     /**
      * The compiler to emulate.
      */
-    readonly intelliSenseMode: "msvc-x64" | "gcc-x64" | "clang-x64";
+    readonly intelliSenseMode: "msvc-x86" | "msvc-x64" | "gcc-x86" | "gcc-x64" | "clang-x86" | "clang-x64";

```

Comment:
please do `npm run compile` to generate the api.d.ts file

Reply:
We also need to update the version in package.json and package-lock.json. Do `npm install` after changing package.json.

3.0.1